src/adapters/discord.js

```diff
@@ -26,23 +27,30 @@ discordAdapter.__INIT__ = function ({ process }) {
         const handle = (context) => {
             const { output, reactions, outputRich } = context;
             // TODO rework!
-            console.log('discord: ', outputRich);
 
             if (outputRich) {
-                if (!isEmpty(reactions)) {
```

Comment:
Why it's deleted?

Reply:
Ou, sorry. I tried how it will be without him and forgot to correct the changes